app/containers/LoginServices.js

```diff
@@ -285,9 +303,26 @@ class LoginServices extends React.PureComponent {
 				onPress = () => this.onPressCas();
 				break;
 			}
+			case 'apple': {
+				onPress = () => this.onPressAppleLogin();
+				break;
+			}
 			default:
 				break;
 		}
+
+		if (name === 'apple') {
```

Comment:
Hide on Android

Reply:
I'm adding `apple` as a supported service only if it's an iOS here: https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/2282/files#diff-6e616dfd576f12234f426a580b065f20R1080